### DISCUSSION TITLE
Show and tell: how you've used censusapi

### DISCUSSION THREAD

**Discussion by hrecht** [hrecht/censusapi#101] [Show and tell]
Hi censusapi community, I'd love to see how you use the package. It's fun but also helps improve future documentation and features. Some of you use the package in ways I've never thought of.

How do you use censusapi? What datasets do you use it with? Has it made anything easier or faster or more reproducible? If you're willing, what organization/company did you use it for? Share links to your code, projects, stories, research, or your testimonials here!

**Comment by szimmer** [comment]
I've used the package to get population totals to adjust weights of a survey. This gets data from the Current Population Survey. Rather than hard coding in a weight adjustment, it can be done and show explicitly where the adjustment comes from.

Full context here: https://github.com/tidy-survey-r/tidy-survey-book/blob/main/04-set-up.Rmd 

Snippet here which gets the voting age, citizen population from CPS:

```
cps_state_in <- getCensus(name = "cps/basic/mar",
                          vintage = 2020,
                          region = "state",
                          vars = c("HRMONTH", "HRYEAR4", 
                                   "PRTAGE", "PRCITSHP", "PWSSWGT"), 
                          key = Sys.getenv("CENSUS_KEY"))

cps_state <- cps_state_in %>%
  as_tibble() %>%
  mutate(across(.cols = everything(),
                .fns = as.numeric))
cps_narrow_resp <- cps_state %>%
  filter(PRTAGE >= 18,
         PRCITSHP %in% c(1:4))
targetpop <- cps_narrow_resp %>%
  pull(PWSSWGT) %>%
  sum()
```
